### ISSUE TITLE
Security vulnerabilities in 1.21.1-alpine

### ISSUE THREAD

**Issue by nijrajgelani-crest** [nginxinc/docker-nginx#563]
Vulnerability scanner report:
```
+---------+------------------+----------+-------------------+---------------+--------------------------------+
| LIBRARY | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |             TITLE              |
+---------+------------------+----------+-------------------+---------------+--------------------------------+
| curl    | CVE-2021-22922   | MEDIUM   | 7.77.0-r1         | 7.78.0-r0     | curl: wrong content via        |
|         |                  |          |                   |               | metalink is not being          |
|         |                  |          |                   |               | discarded                      |
+         +------------------+          +                   +               +--------------------------------+
|         | CVE-2021-22923   |          |                   |               | curl: Metalink download sends  |
|         |                  |          |                   |               | credentials                    |
+         +------------------+----------+                   +               +--------------------------------+
|         | CVE-2021-22924   | LOW      |                   |               | curl: bad connection reuse due |
|         |                  |          |                   |               | to flawed path name checks     |
+         +------------------+          +                   +               +--------------------------------+
|         | CVE-2021-22925   |          |                   |               | curl: Incorrect fix for        |
|         |                  |          |                   |               | CVE-2021-22898 TELNET stack    |
|         |                  |          |                   |               | contents disclosure            |
+---------+------------------+----------+                   +               +--------------------------------+
| libcurl | CVE-2021-22922   | MEDIUM   |                   |               | curl: wrong content via        |
|         |                  |          |                   |               | metalink is not being          |
|         |                  |          |                   |               | discarded                      |
+         +------------------+          +                   +               +--------------------------------+
|         | CVE-2021-22923   |          |                   |               | curl: Metalink download sends  |
|         |                  |          |                   |               | credentials                    |
+         +------------------+----------+                   +               +--------------------------------+
|         | CVE-2021-22924   | LOW      |                   |               | curl: bad connection reuse due |
|         |                  |          |                   |               | to flawed path name checks     |
+         +------------------+          +                   +               +--------------------------------+
|         | CVE-2021-22925   |          |                   |               | curl: Incorrect fix for        |
|         |                  |          |                   |               | CVE-2021-22898 TELNET stack    |
|         |                  |          |                   |               | contents disclosure            |
+---------+------------------+----------+-------------------+---------------+--------------------------------+
```

These seem to have already been fixed in alpine:3.14.0.

**Comment by IdamkinI** [comment]
I'm also getting next vulnerability for `apk-tools` with `clair` scanner:
```
 +------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
 | Unapproved | Unknown CVE-2021-36159 | apk-tools    | 2.12.5-r1       |                                                               |
 |            |                        |              |                 | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-36159 |
 +------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
```

Note that both issues can be fixed by updating packages with:
```
RUN apk add --update --no-cache 'apk-tools==2.12.7-r0'
RUN apk add --update --no-cache 'curl==7.78.0-r0'
```

**Comment by yosifkit** [comment]
Same comment as https://github.com/nginxinc/docker-nginx/issues/564#issuecomment-894539878

**Comment by thresheek** [comment]
The fixes are now a part of `nginx:1.21.1-alpine` since a few days ago, closing this one.
